subgraphs/tunnel-deposits-subgraph/subgraph.yaml

```diff
@@ -0,0 +1,29 @@
+specVersion: 1.0.0
+indexerHints:
+  prune: auto
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: L1StandardBridge
+    network: sepolia
+    source:
+      abi: L1StandardBridge
+      address: '0xc94b1BEe63A3e101FE5F71C80F912b4F4b055925'
+      startBlock: 5294649
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      entities:
+        - Deposit
+      abis:
+        - name: L1StandardBridge
+          file: ./abis/L1StandardBridge.json
+      eventHandlers:
+        - event: ETHDepositInitiated(indexed address,indexed address,uint256,bytes)
+          handler: handleETHDeposit
+        - event: ERC20DepositInitiated(indexed address,indexed address,indexed
+            address,address,uint256,bytes)
```

Comment:
The Blockscout explorer uses `TransactionDeposited` to identify all the deposits. Let's analyze using that event instead.

https://github.com/hemilabs/blockscout/blob/5b1e9b1c8883cfa926674a8993f5f7fc304c1a4d/apps/indexer/lib/indexer/fetcher/optimism/deposit.ex#L37

Reply:
I went for Deposit Initiated as [this is the Event used by the OP SDK](https://github.com/ethereum-optimism/optimism-legacy/blob/8205f678b7b4ac4625c2afe351b9c82ffaa2e795/packages/sdk/src/adapters/eth-bridge.ts#L36), which is the package we based our code from

Reply:
Note that the `TransactionDeposited` event is missing some information that the other event does have, particularly the L1, L2 token addresses and the amount.

See

```

ETHDepositInitiated (index_topic_1 address _l1Token, index_topic_2 address _l2Token, index_topic_3 address _from, address _to, uint256 _amount, bytes _data)

TransactionDeposited (index_topic_1 address from, index_topic_2 address to, index_topic_3 uint256 version, bytes opaqueData)
```

Perhaps we can parse them from the TransactionInput, but maybe that means extra work as we need to parse the input